packages/discord.js/src/managers/GuildScheduledEventManager.js

```diff
@@ -185,25 +183,22 @@ class GuildScheduledEventManager extends CachedManager {
    * @param {GuildScheduledEventEditOptions} options Options to edit the guild scheduled event
    * @returns {Promise<GuildScheduledEvent>}
    */
-  async edit(guildScheduledEvent, options) {
+  async edit(guildScheduledEvent, {
+    privacyLevel,
+    entityType,
+    channel,
+    status,
+    name,
+    scheduledStartTime,
+    description,
+    scheduledEndTime,
+    entityMetadata,
+    reason,
+    image,
+  }) {
     const guildScheduledEventId = this.resolveId(guildScheduledEvent);
     if (!guildScheduledEventId) throw new Error('GUILD_SCHEDULED_EVENT_RESOLVE');
 
-    if (typeof options !== 'object') throw new TypeError('INVALID_TYPE', 'options', 'object', true);
```

Comment:
This may be nothing, as I may have missed some important note about discord.js' policy on handling runtime type issues going forward, but I figured I'd bring this up just in case:

Isn't removing this type check (and similar ones in this PR) a functional regression? If someone (who's not using any static type-checking tools) passes a non-object in for `options`, I'd assume the runtime errors would be about the destructured properties being `undefined`, as opposed to the previous easy-to-understand `TypeError` against the parameter as a whole.